src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastHelpers.cs

```diff
@@ -33,6 +37,19 @@ internal static void ThrowInvalidCastException(object fromType, void* toTypeHnd)
         [MethodImpl(MethodImplOptions.InternalCall)]
         private static extern void WriteBarrier(ref object? dst, object? obj);
 
+        internal static ref byte GetRefAny(IntPtr clsHnd, TypedReference typedByRef)
+        {
+            // <TODO>@TODO right now we check for precisely the correct type.
+            // do we want to allow inheritance?  (watch out since value
+            // classes inherit from object but do not normal object layout).</TODO>
```

Comment:
This TODO is there since coreclr's initial commit. Is it still relevant?

Reply:
I would remove this. It is something we will discover if needed through testing.

Reply:
```suggestion
```